client/README.md

```diff
@@ -1,9 +1,188 @@
-InfluxDB Go Client Library
-============
+# InfluxDB Client
 
-Please refer to [http://godoc.org/github.com/influxdb/influxdb/client](http://godoc.org/github.com/influxdb/influxdb/client)
+## Description
+
+A Go client library written and maintain by the **InfluxDB** team.
+This package provides convenience functions to read and write time series data.
+It uses the HTTP protocol to communicate with your **InfluxDB** cluster.
+
+
+## Getting Started
+
+### Connecting To Your Database
+
+Connecting to an **InfluxDB** database is straight forward. You will need a host
+name, a port and the cluster credentials. The default port is 8086 and the default
+credentials is root for both username and password. You can customize these settings
+to your specific installation via the **InfluxDB** configuration file.
+
+```go
+package main
+
+import "github.com/influxdb/influxdb/client"
+
+const (
+	MyHost        = "localhost"
+	MyPort        = 8086
+	MyDB          = "square_holes"
+	MyMeasurement = "shapes"
+	MyUser        = "root"
```

Comment:
It would be nice to see an example with and without credentials being sent.  As long as we clarify as stated above that we don't have a default username/password.  I can't remember if we updated our docs about how to bring up an authenticated server, but if we did, we could link to those docs as well for people to see how that is done, as it would probably be helpful in understanding the different server setups.